_site/content/posts/modules/cryptocurrencies/cryptolive.md

```diff
@@ -83,6 +82,6 @@ Values: `true`, `false`.
 `position` <br />
 Defines where in the grid this module's widget will be displayed. <br />
 
-`refreshInterval` <br />
+`updateInterval` <br />
```

Comment:
I'm confused by this now. All widgets have a `refreshInterval` (ex https://wtfutil.com/posts/modules/clocks/) that defines how often they refresh their data. You've changed the name of it here to something different but with the same description...?